pkgs/development/tools/move/move-cli/default.nix

```diff
@@ -0,0 +1,67 @@
+{ fetchFromGitHub
+, lib
+, rustPlatform
+, pkg-config
+, openssl
+, zlib
+, stdenv
+, darwin
+, git
+}:
+
+let
+  common =
+    { buildFeatures ? [ ]
+    }:
+    rustPlatform.buildRustPackage rec {
+      pname = "move";
+      version = "f20499851934cd51f81b390954a292ca1bd419b8";
```

Comment:
I think this should be in `unstable-YYYY-mm-dd` format - https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming

Reply:
updated